front/tests/pages/op-route-page-model.ts

```diff
@@ -0,0 +1,327 @@
+import { expect, type Locator, type Page } from '@playwright/test';
+
+import enTranslations from '../../public/locales/en/operationalStudies/manageTrainSchedule.json';
+import frTranslations from '../../public/locales/fr/operationalStudies/manageTrainSchedule.json';
+import { clickWithDelay } from '../utils';
+
+class RoutePage {
+  readonly page: Page;
+
+  readonly pathfindingNoState: Locator;
+
+  readonly noOriginChosen: Locator;
+
+  readonly noDestinationChosen: Locator;
+
+  readonly searchByTrigramButton: Locator;
+
+  readonly searchByTrigramContainer: Locator;
+
+  readonly searchByTrigramInput: Locator;
+
+  readonly searchByTrigramSubmit: Locator;
+
+  readonly resultPathfindingDone: Locator;
+
+  readonly originInfo: Locator;
+
+  readonly destinationInfo: Locator;
+
+  readonly viaInfo: Locator;
+
+  readonly originDeleteButton: Locator;
+
+  readonly destinationDeleteButton: Locator;
+
+  readonly viaDeleteButton: Locator;
+
+  readonly addWaypointsButton: Locator;
+
+  readonly reverseItineraryButton: Locator;
+
+  readonly deleteItineraryButton: Locator;
+
+  readonly droppedWaypoints: Locator;
+
+  readonly waypointSuggestions: Locator;
+
+  readonly viaModal: Locator;
+
+  readonly closeViaModalButton: Locator;
+
+  readonly missingParamMessage: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.pathfindingNoState = page.getByTestId('pathfinding-no-state');
+    this.noOriginChosen = page.getByTestId('no-origin-chosen-text');
+    this.noDestinationChosen = page.getByTestId('no-destination-chosen-text');
+    this.searchByTrigramButton = page.getByTestId('rocket-button');
+    this.searchByTrigramContainer = page.getByTestId('type-and-path-container');
+    this.searchByTrigramInput = page.getByTestId('type-and-path-input');
+    this.searchByTrigramSubmit = page.getByTestId('submit-search-by-trigram');
+    this.resultPathfindingDone = page.getByTestId('result-pathfinding-done');
+    this.originInfo = page.getByTestId('origin-op-info');
+    this.destinationInfo = page.getByTestId('destination-op-info');
+    this.viaInfo = page.getByTestId('via-op-info');
+    this.originDeleteButton = page.getByTestId('delete-origin-button');
+    this.destinationDeleteButton = page.getByTestId('delete-destination-button');
+    this.viaDeleteButton = page.getByTestId('delete-via-button');
+    this.addWaypointsButton = page.getByTestId('add-waypoints-button');
+    this.reverseItineraryButton = page.getByTestId('reverse-itinerary-button');
+    this.deleteItineraryButton = page.getByTestId('delete-itinerary-button');
+    this.droppedWaypoints = page.getByTestId('dropped-via-info');
+    this.waypointSuggestions = page.locator('.suggested-via-clickable');
+    this.viaModal = page.locator('.manage-vias-modal');
+    this.closeViaModalButton = page.getByLabel('Close');
+    this.missingParamMessage = page.getByTestId('missing-params-info');
+  }
+
+  // Gets the name locator of a waypoint suggestion.
```

Comment:
```suggestion
  // Get the name locator of a waypoint suggestion.
```

same for every similar comment

Reply:
sorry but only one has been changed